repo2docker/buildpacks/base.py

```diff
@@ -608,7 +609,33 @@ def _filter_tar(tar):
         for fname in ("repo2docker-entrypoint", "python3-login"):
             tar.add(os.path.join(HERE, fname), fname, filter=_filter_tar)
 
-        tar.add(".", "src/", filter=_filter_tar)
+        exclude = []
+
+        for ignore_file_name in [".dockerignore", ".containerignore"]:
+            if os.path.exists(ignore_file_name):
```

Comment:
Let's make sure we respect the `binder` dir overrides by using `self.binder_path(ignore_file_name)`, sot he top-level file can be ignored.

Reply:
Thanks for pointing that out ! I had forgotten about that configurable part.